awx/main/expect/isolated_manager.py

```diff
@@ -198,9 +198,20 @@ def check(self, interval=None):
                 for event in set(os.listdir(events_path)) - self.handled_events:
                     path = os.path.join(events_path, event)
                     if os.path.exists(path):
-                        event_data = json.load(
-                            open(os.path.join(events_path, event), 'r')
-                        )
+                        try:
+                            event_data = json.load(
+                                open(os.path.join(events_path, event), 'r')
+                            )
+                        except json.decoder.JSONDecodeError:
+                            # This means the event we got back isn't valid JSON
```

Comment:
@matburt before runner, the job events we wrote were written as partial files and *moved* into the final file name to provide atomic writes (so we avoided scenarios where we rsynced back files which were still being written).  I'm seeing evidence that this behavior is *not* preserved in runner:

<img width="365" alt="image" src="https://user-images.githubusercontent.com/214912/55240016-43154d00-520e-11e9-9f2c-8bca8dfc2fff.png">

...so I'm employing some defensive programming for now to work around this.